sv.c

```diff
@@ -17735,6 +17719,43 @@ Perl_report_uninit(pTHX_ const SV *uninit_sv)
     GCC_DIAG_RESTORE_STMT;
 }
 
+/* This helper function for Perl_sv_setsv_flags is as cold as they come.
```

Comment:
I'm not so sure about the function name, it sounds like a version of sv_setsv_flags() that croaks in some circumstances rather than a function sv_setsv_flags() calls when it needs to croak.

maybe croak_for_sv_setsv() (or with _flags) but better ideas welcome.

Reply:
Yes, good point. Will sleep on it.

Reply:
Ok, I went with `S_croak_sv_setsv_flags`. It seemed a reasonable fit with the existing `croak_*` names.